src/subscription_manager/gui/data/glade/installed.glade

```diff
@@ -3,6 +3,19 @@
   <!-- interface-requires gtk+ 2.10 -->
   <!--
  interface-naming-policy project-wide -->
+  <object class="GtkAdjustment" id="adjustment1">
+    <property name="upper">396</property>
+    <property name="step_increment">39.6</property>
+    <property name="page_increment">356.4</property>
```

Comment:
As best I can tell, the scrolled windows don't actually seem to care what these values are, just that they have an Adjustment attached.

Something like
``` xml
<object class="GtkAdjustment" id="adjustment1">
    <property name="upper">100</property>
    <property name="step_increment">1</property>
    <property name="page_increment">10</property>
 </object>
```
... should work just as well and be a a little less confusing. I think the number in the old glade versions just
correspond to whatever glade thought the adjustment values were when the project was saved.